lib/govuk_tech_docs/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukTechDocs
-  VERSION = "2.2.1".freeze
+  VERSION = "2.2.2".freeze
```

Comment:
I think we tend to do releases separately in this repo, rather than making a change and releasing in the same PR.

Reply:
Thanks - will update